### ISSUE TITLE
2.9.14 patch release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3572]
- ~#3519~
- #3545
- #3589
- #3584

## Pick list
```
git cherry-pick 747de76b53d19f96ce93e4bbf3c9aa28f4f00c72
git cherry-pick aec34ffd9fe24c938593454acdd58105a348e3cb
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.9.13..HEAD --pretty='format:- %s'`
```
*Important

- Close #3588 Allow newly enabled module config to be overridden. (#3589)
- Fixes #3544 AZBackgroundMediaFormatter Breaks with Missing Files (#3545)
- #3581: Apply updated coding standards to 2.9.x branch. (#3584)
- Prepare 2.9.x branch for new minor release branch 2.11.x (#3527)
- Back to dev.
```

**Comment by joeparsons** [comment]
Completed: https://github.com/az-digital/az_quickstart/releases/tag/2.9.14
